test/recipes/70-test_tls13kexmodes.t

```diff
@@ -29,7 +29,7 @@ plan skip_all => "$test_name needs TLSv1.3 enabled"
     if disabled("tls1_3") || (disabled("ec") && disabled("dh"));
 
 plan skip_all => "$test_name needs EC enabled"
-    if disabled("ec");
+    if disabled("ec") || disabled("ecx");
```

Comment:
This does not look right. The test should work with no-ecx.

Reply:
yes.. But it does fail even though I don't find where it uses ecx.
Same thing happened in test_tls13hrr test_tls13psk test_tls13messages,
let me know if you have any idea and i will continue to investigate it.